### PR TITLE
Update query_entities() samples

### PR DESCRIPTION
**Pull request by YalinLi0312**
For issue: https://github.com/Azure/azure-sdk-for-python/issues/20282

Add headers in query_entities() to get a cleaner JSON response.

### CODE REVIEW

File: sdk/tables/azure-data-tables/samples/sample_query_table.py [hunk 1]
```diff
@@ -139,7 +107,5 @@ def clean_up(self):
     try:
         stq.insert_random_entities()
         stq.sample_query_entities()
-        stq.sample_query_entities_multiple_params()
-        stq.sample_query_entities_values()
```
Comment (annatisch) [inline sdk/tables/azure-data-tables/samples/sample_query_table.py:143]: Why were these removed?

File: sdk/tables/azure-data-tables/samples/sample_query_table.py [hunk 2]
```diff
@@ -68,59 +68,27 @@ def sample_query_entities(self):
...
-                parameters = {"name": "marker"}
-                name_filter = "Name eq @name"
-                queried_entities = table_client.query_entities(
-                    query_filter=name_filter, select=["Brand", "Color"], parameters=parameters
+                print("Entities with name: marker and brand: Crayola")
+                queried_entities1 = table_client.query_entities(
+                    query_filter={"name": "marker", "brand": "Crayola"},
+                    select=["Brand", "Color"],
+                    parameters="Name eq @name and Brand eq @brand",
+                    headers ={"Accept" : "application/json;odata=nometadata"}
```
Comment (annatisch) [inline sdk/tables/azure-data-tables/samples/sample_query_table.py:79]: We should not add this to an existing sample - let's leave the existing samples untouched as adding this header should be very uncommon.
I would add a new method called something like `sample_query_entities_without_metadata` and add it the bottom of the file.

File: sdk/tables/azure-data-tables/samples/sample_query_table.py [hunk 3]
```diff
@@ -68,59 +68,27 @@ def sample_query_entities(self):
...
             try:
-                parameters = {"name": "marker"}
-                name_filter = "Name eq @name"
-                queried_entities = table_client.query_entities(
-                    query_filter=name_filter, select=["Brand", "Color"], parameters=parameters
+                print("Entities with name: marker and brand: Crayola")
+                queried_entities1 = table_client.query_entities(
+                    query_filter={"name": "marker", "brand": "Crayola"},
+                    select=["Brand", "Color"],
+                    parameters="Name eq @name and Brand eq @brand",
```
Comment (annatisch) [inline sdk/tables/azure-data-tables/samples/sample_query_table.py:78]: This is incorrect - the `query_filter` and `parameters` values are backwards.

File: sdk/tables/azure-data-tables/samples/sample_query_table.py (hunk 1 above)
Comment (YalinLi0312) [reply sdk/tables/azure-data-tables/samples/sample_query_table.py:143]: I moved these three functions into one.

File: sdk/tables/azure-data-tables/samples/sample_query_table.py (hunk 2 above)
Comment (YalinLi0312) [reply sdk/tables/azure-data-tables/samples/sample_query_table.py]: Updated, added a new function for `no metadata` headers.

**Review by annatisch — APPROVED** [review]
